hivemq-edge/src/main/java/com/hivemq/edge/modules/adapters/simulation/SimulationProtocolAdapterInformation.java

```diff
@@ -109,4 +112,20 @@ public ProtocolAdapterCategory getCategory() {
             return null;
         }
     }
+
+
+    @Override
+    public @NotNull Class<? extends Tag> tagConfigurationClass() {
+        return null; //TODO what would be correct here?
```

Comment:
good question

Reply:
if we set the default to return null it should return @Nullable

Reply:
Changed the comment :)